### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by EtDu**

### CODE REVIEW

File: packages/core/src/makePrepareRealmGlobalFromConfig.js [hunk 1]
```diff
@@ -82,11 +82,11 @@ function makePrepareRealmGlobalFromConfig () {
...
-      return origFunction(...src).bind(moduleRealmGlobal)
-    }
-    Object.defineProperties(newFunction, Object.getOwnPropertyDescriptors(origFunction))
-    moduleRealmGlobal.Function = newFunction
+    // const origFunction = moduleRealmGlobal.Function
+    // const newFunction = function (...src) {
+    //   return origFunction(...src).bind(moduleRealmGlobal)
+    // }
+    // Object.defineProperties(newFunction, Object.getOwnPropertyDescriptors(origFunction))
+    // moduleRealmGlobal.Function = newFunction
```
Comment (kumavis) [inline packages/core/src/makePrepareRealmGlobalFromConfig.js]: remove plz

**Review by kumavis — CHANGES_REQUESTED** [review]
and fix ci

File: packages/core/src/makePrepareRealmGlobalFromConfig.js (hunk 1 above)
Comment (EtDu) [reply packages/core/src/makePrepareRealmGlobalFromConfig.js]: okeeee

**Review by kumavis — APPROVED** [review]
what could possibly 🐌

**Pull request merged**